process faults {3: ({0}, 690571153)}
network faults {}
{9: ("35C61810E9E8B702720195B4FFA7534D4F71B7B934EBFD276413BEC12DCCC30F", {3, 0, 4, 1, 6, 2, 5}), 8: ("24D9974E1C2E13E2CA83E5D251CAF2CE27A0B7C14FAB1902AC07E75F0CDC33C8", {2, 1, 0, 3, 6, 4, 5}), 5: ("7F7D3728BCD3EF8D15C04B0622440AC987DC81ABBC844503CBB9D29490082A43", {6, 3, 2, 4, 1, 5, 0}), 6: ("23918BCDCF2EA28C306195C25F1DE7DF3EE33CE4F129590C11302B9E63778D62", {0, 5, 2, 3, 6, 4, 1}), 10: ("1BFBBC48EB81E59D002EF85466CDA696DEF6D72BD0560F92274ADC9ECE1C9007", {1, 2, 5, 0, 6, 3, 4}), 7: ("60A8545DCE4E25A39AA042917D60C4FE98BA3211A23DB86B3477069BA19E05FC", {1, 0, 6, 2, 3, 4, 5})}
7
reason: all committed
done!
